src/pages/overview/document-generation-api/dynamictableconstructs.md

```diff
@@ -132,3 +132,55 @@ JSON representation of the input data:
 
 - {{Conversion.Rate:**discard-if(expr($sum(Conversion.Rate)!= 100))**}}% tag lets the engine discard the particular column if condition provided in the **expr** construct evaluates to true.
 
+
+## Dynamic table rows
+Discard a row or set of rows in a table from the final generated document.
+
+### Discard row if condition evaluates to true
+Tag: **discard-row-if(expr(condition),context)**
```

Comment:
Explain tag with an example, instead of writing Tag:....

Reply:
This is done to explicitly tell which is the new thing we are adding here. Usually a user want to copy a tag in order to use it but we are not providing like this in other constructs, hence added this for the ease.

explained that here https://github.com/AdobeDocs/pdfservices-api-documentation/pull/432/files#diff-a71e6e9ad91f0d070ee7d60f4593bab1d0c57853ab12ab861b62c8be62bf8307R144
https://github.com/AdobeDocs/pdfservices-api-documentation/pull/432/files#diff-a71e6e9ad91f0d070ee7d60f4593bab1d0c57853ab12ab861b62c8be62bf8307R144